src/components/login/index.js

```diff
@@ -0,0 +1,143 @@
+import {
+  Box,
+  Button,
+  Checkbox,
+  Container,
+  Divider,
+  FormControl,
+  FormLabel,
+  Heading,
+  HStack,
+  Input,
+  Link,
+  Stack,
+  Text,
+  useBreakpointValue,
+  useColorModeValue,
+} from '@chakra-ui/react'
+import axios from 'axios'
+import { useRouter } from 'next/router'
+import { useTranslation } from 'next-i18next'
+import * as React from 'react'
+import { useState } from 'react'
+import { useForm } from 'react-hook-form'
+
+import { useUser } from '~hooks'
+
+import { OAuthButtonGroup } from './oauth-button-group'
+import { PasswordField } from './password-field'
+
+export const Login = () => {
+  const {
+    register,
+    handleSubmit,
+    reset,
+    formState: { errors },
+  } = useForm()
+  const [errorMessage, setErrorMessage] = useState('')
+  const { t } = useTranslation()
+  useUser('/profile', true)
+
+  const router = useRouter()
+
+  const handleSubmitSign = async data => {
+    const body = {
+      identifier: data.email,
+      password: data.password,
+    }
+
+    try {
+      const resp = await axios.post('/api/auth/login', body)
+      if (resp?.data?.error) {
+        setErrorMessage(resp?.data?.error?.message)
+        setTimeout(() => {
+          setErrorMessage('')
+          reset()
+        }, 2000)
+      } else {
+        reset()
+        router.push('/')
+      }
+    } catch (error) {
+      if (error?.response?.data?.error?.message) {
+        setErrorMessage(eerror?.response?.data?.error?.message)
+        setTimeout(() => {
+          setErrorMessage('')
+        }, 2000)
+      } else {
+        console.error('An unexpected error happened:', error)
+      }
+    }
+  }
+
+  return (
+    <Container maxW='lg' py={{ base: '12', md: '24' }} px={{ base: '0', sm: '8' }}>
+      <Stack spacing='8'>
+        <Stack spacing='6'>
+          <Stack spacing={{ base: '2', md: '3' }} textAlign='center'>
+            <Heading size={useBreakpointValue({ base: 'xs', md: 'sm' })}>{t('login.sign-in-header.title')}</Heading>
+            <HStack spacing='1' justify='center'>
+              <Text color='muted'>{t('login.sign-in-header.text')}</Text>
+              <Button variant='link' as={Link} href='/user/register' colorScheme='blue'>
+                {t('login.sign-in-header.button')}
+              </Button>
+            </HStack>
+          </Stack>
+        </Stack>
+        <Box
+          py={{ base: '0', sm: '8' }}
+          px={{ base: '4', sm: '10' }}
+          bg={useBreakpointValue({ base: 'transparent', sm: 'bg-surface' })}
```

Comment:
We use `useBreakpointValue` when Chakra's responsive props approach con not be applied to some props.
And make sure whether we have `bg-surface` color in the theme. Otherwise I would remove these lines.

```suggestion
          bg={{ base: 'transparent', sm: 'bg-surface' }}
```